README.md

```diff
@@ -126,6 +126,14 @@ Similarly, you can customize your publish directory in your `netlify.toml` file:
 
 Read more about [Netlify's build settings](https://docs.netlify.com/configure-builds/get-started/#basic-build-settings) in our docs.
 
+## Image handling
```

Comment:
do you wanna add an anchor to this with the other section headings at the top? :)